data/json/snippets/effect_on_conditions.json

```diff
@@ -189,84 +189,84 @@
+      "a gentle voice saying \"Can you feel it?  The unmaking in the making?  But the change isn't against you.  You need to face it to ride the tide.\"",
       "a gentle voice saying \"Your existence has been frowned upon by many eyebrows.  But you too have a pair of eyebrows to frown back.\"",
       "a deep voice saying \"ᛁᛟᚢᚱ ᛁᚾᚹᛁᛏᚨᛏᛁᛟᚾ ᛏᛟ ᚹᚨᛚᚺᚨᛚᛚᚨ ᚺᚨᛊ ᛒᛖᛖᚾ ᚱᛖᚹᛟᚲᛖᛞ\"",
       "a gentle voice saying \"Fiery the angels fall.  Are you there yet?  At the bottom so low that the only way is up?\"",
       "a gentle voice saying \"Sometimes dead is better.  But surely you wish to walk a different path?  So why do you stray from the road?\"",
       "a gentle voice saying \"Tell me what thy lordly name is?  Tell me, tell me, I implore!\"",
       "a gentle voice saying \"I have come to lead you to the other shore; into eternal darkness; into fire and into ice.\"",
-      "a gentle voice saying \"The cancer devours the flesh of this world.  You're next in line, but there is a path out of this.  Just walk the way.\""
+      "a gentle voice saying \"The cancer devours the flesh of this world.  You're next in line, but there is a path out of this.  Just walk the way.\"",
+      "a gentle voice saying \"Please come out.  I can't do this alone.  Please?\""
     ]
   },
   {
     "type": "snippet",
     "category": "PORTAL_STORM_VOICES_2",
     "text": [
-      "a voice saying \"All your leaders have walked this path and have chosen salvation.  It was a wise decision.  The right decision.\"",
-      "a voice saying \"The beyond can give you what you have always craved, purpose.\"",
+      "a voice saying \"Everyone you knew has walked this path and has chosen salvation.  It was a wise decision.  The right decision.\"",
+      "a voice saying \"The beyond can give you what you have always craved.  Purpose.\"",
       "a voice saying \"Never before has the path to freedom been so straight.  Never before has it been so close.\"",
       "a voice saying \"Remember when you were young and dreams could still come true?  Outside they still can.\"",
-      "a voice saying \"The wonders in the sky are beyond your imagination.  Can you even guess at what you are missing right now?\"",
-      "a voice saying \"All your life you have been oppressed by cruel physics, come outside and see freedom.\"",
-      "a voice saying \"What you may think you may lose is but a speck of what you will gain.  Walk outside and embrace the infinity.\"",
-      "a voice saying \"Do you want to live the rest of your life hiding like a coward or come out and face your fears?\"",
+      "a voice saying \"The wonders in the sky are beyond your imagination.  Can you even imagine what you're missing right now?\"",
+      "a voice saying \"All your life you have been oppressed by what you thought were physics.  You're so close to freedom.\"",
```

Comment:
Minor detail but in my head version at least its not that you were wrong about physics but that in fact physics have changed or are beginning to break.